### PR TITLE
[Tests-Only] Fixed failing GUI tests for VFS

### PR DESCRIPTION
**Pull request by Talank**
I have used screenshot comparison to check if VFS is enabled or not. The tolerance of 16 has been set so that it ignores difference in background color of the object to be compare.

### Related Issue
Fixes https://github.com/owncloud/client/issues/9303

### How this fix works?
Since the button to disable VFS is not available, we had to do the visual test. Also, the content test was not reliable so, I used the screenshot testing. In order to focus on the availability if the element shown in the picture, I have created a positive mask around the text `Virtual file support is enabled.` Also, threshold and tolerance of 4.5 is used because in drone, the difference is less than 4.5 and failing difference is more than 4.5 (about 5.8%).
Screenshot from 2021-12-20 15-07-24

### CODE REVIEW

**Comment by Talank** [comment]
Unrelated error in drone https://drone.owncloud.com/owncloud/client/10112/5/14 I am restarting the build. Since there is only one error, the changes in this PR should fix the failing VFS test.
```
{
--
8 | "Error Details": "Exception: Step execution through test middleware failed. Error: AssertionError [ERR_ASSERTION]: Expected share with user Alice and resource /textfile0.txt is present!\n[{\"id\":\"19\",\"share_type\":3,\"uid_owner\":\"Alice\",\"displayname_owner\":\"Alice Hansen\",\"permissions\":1,\"stime\":1639997119,\"parent\":null,\"expiration\":null,\"token\":\"9S7ge3ehjrSGPfN\",\"uid_file_owner\":\"Alice\",\"displayname_file_owner\":\"Alice Hansen\",\"additional_info_owner\":null,\"additional_info_file_owner\":null,\"path\":\"/textfile0.txt\",\"mimetype\":\"text/plain\",\"storage_id\":\"home::Alice\",\"storage\":35,\"item_type\":\"file\",\"item_source\":201,\"file_source\":201,\"file_parent\":199,\"file_target\":\"/textfile0.txt\",\"name\":\"Public-link\",\"url\":\"http://owncloud/index.php/s/9S7ge3ehjrSGPfN\",\"mail_send\":0,\"attributes\":null}]\n    at StepDef.action (/drone/src/middleware/src/stepDefinitions/publicLinkContext.js:57:14)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async /drone/src/middleware/src/app.js:66:7",
9 | "Feature": "Sharing",
10 | "Feature File": "tst_sharing",
11 | "Scenario": "sharing of a file by public link and deleting the link",
12 | "Test Step": "Then as user \"Alice\" the file \"/textfile0.txt\" should not have any public link on the server"
13 | }
```

File: test/gui/tst_vfs/test.feature [hunk 1]
```diff
@@ -9,12 +9,10 @@ Feature: Enable/disable virtual file support
...
-        Then the "Disable virtual file support..." button should be available
+        Then VFS enabled baseline image should match the default screenshot
 
 
     Scenario: Disable VFS
         Given user "Alice" has been created on the server with default attributes and without skeleton files
         And user "Alice" has set up a client with default settings
-        And the user has enabled virtual file support
-        When the user disables virtual file support
-        Then the "Enable virtual file support (experimental)..." button should be available
```
Comment (saw-jan) [inline test/gui/tst_vfs/test.feature:20]: So can we also remove the respective step definitions?

Comment (Talank) [reply test/gui/tst_vfs/test.feature:20]: removed.

**Review by saw-jan — APPROVED** [review]
LGTM

**Comment by Talank** [comment]
This one is strange. Our test result parser is not being able to figure out which test has failed. https://drone.owncloud.com/owncloud/client/10117/5/14

**Pull request merged**